common/scala/src/main/scala/whisk/core/containerpool/logging/SplunkLogStore.scala

```diff
@@ -112,11 +112,7 @@ class SplunkLogStore(
       Map(
         "exec_mode" -> "oneshot",
         "search" -> search,
-        "output_mode" -> "json",
-        "earliest_time" -> activation.start.toString, //assume that activation start/end are UTC zone, and splunk events are the same
-        "latest_time" -> activation.end
-          .plusSeconds(5) //add 5s to avoid a timerange of 0 on short-lived activations
-          .toString)).toEntity
+        "output_mode" -> "json")).toEntity
```

Comment:
I think the "no results" case also needs to be handled, regardless of whether the query is correct. 

It is still not clear why the timestamp queries wouldn't work - my concern is that this could be problematic depending on the config of the splunk index; do you have example logs and queries where this fails?